packages/sui-bundler/webpack.config.client.dev.js

```diff
@@ -49,6 +50,11 @@ const webpackConfig = {
   entry: {
     app: [`webpack-hot-middleware/client?path=${CDN}__webpack_hmr`, MAIN_ENTRY_POINT]
   },
+  cache: {
+    type: 'filesystem',
+    cacheDirectory,
+    compression: 'gzip'
```

Comment:
Could we use Brotli to improve compression??

Reply:
good point

Reply:
Yes! changes made. Thank you guys